### PR TITLE
Magento_GroupedProduct: avoid using deprecated escape* methods from A…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/GroupedProduct/view/adminhtml/templates/catalog/product/composite/fieldset/grouped.phtml [hunk 1]
```diff
@@ -6,40 +6,44 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
 
-<?php /* @var $block \Magento\GroupedProduct\Block\Adminhtml\Product\Composite\Fieldset\Grouped */ ?>
+<?php
+/**
+ * @var $block \Magento\GroupedProduct\Block\Adminhtml\Product\Composite\Fieldset\Grouped
```
Comment (ihor-sviziev) [inline app/code/Magento/GroupedProduct/view/adminhtml/templates/catalog/product/composite/fieldset/grouped.phtml:11]: it seems like you missed adding escaper here

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/GroupedProduct/view/adminhtml/templates/catalog/product/composite/fieldset/grouped.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/GroupedProduct/view/adminhtml/templates/catalog/product/composite/fieldset/grouped.phtml:11]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests EE

**Pull request closed without merging**